istioctl/pkg/install/verify.go

```diff
@@ -182,13 +184,17 @@ func verifyPostInstall(enableVerbose bool, istioNamespaceFlag string,
 			// usual conversion not available.  Convert unstructured to string
 			// and ask operator code to unmarshal.
 
-			un.SetCreationTimestamp(meta_v1.Time{}) // UnmarshalIstioOperator chokes on these
+			cleanupForGogo(un)
```

Comment:
is this gogo specific? will it work if gogo is replaced? if yes, i suggest removing gogo from the name and just do whatever is needed.

Reply:
The function is a work around for what is either a bug in Gogo or in Kubernetes.  This prevents `Error: could not load IstioOperator from cluster: could not unmarshal: json: cannot unmarshal string into Go value of type map[string]json.RawMessage`.  I suspect it is related to https://github.com/istio/istio/issues/26010 but I haven't proved that yet.